cassandra/cluster.py

```diff
@@ -139,6 +141,9 @@ def _is_gevent_monkey_patched():
     except ImportError:
         from cassandra.io.asyncorereactor import AsyncoreConnection as DefaultConnection  # NOQA
 
+# If true tablet experimental feature is enabled, its interface and use may change
+EXPERIMENTAL_TABLETS = os.environ.get('EXPERIMENTAL_TABLETS', '').lower() == 'true'
```

Comment:
I wanted to find some way to do experimental setting using the most general approach to be able to turn it on/off for the driver itself as a whole. I do not think there is a need to have the possibility to create 2 `Cluster` objects, one with tablets and one without, because tablets are set on the keyspace level. In the gocql I used flag on the cluster level and it was consider not be perfect approach. I understand that this solution is not really good either. Maybe there is a different way to enable experimental feature for the whole library (for example during installation) in python? If not, I will change it to the solution you suggested.

Reply:
Afaik Python doesn't have anything resembling e.g. Rust's crate features.
This PR already adds a parameter to Cluster constructor, so we get disadvantages related to that.
I think we should go with Cluster constructor parameters (one boolean, which enables / disables tablets support, second one for refresh interval), but
- Prefix their name with `experimental_`, same goes for field name.
- Add comment constructor parameters and related field informing that they are experimental and may be changed / removed.

Reply:
Okay, I will do this that way

Reply:
Done